package.json

```
@@ -211,7 +211,8 @@
     "fsevents": "^2.3.2"
   },
   "resolutions": {
-    "**/ua-parser-js": "<=0.7.28"
+    "**/ua-parser-js": "<=0.7.28",
+    "postcss": "8.4.33"
```

we happened to bump postcss already
```suggestion
    "**/ua-parser-js": "<=0.7.28"
```